packages/design-system/src/__theme__/getIconByKind.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
Can you put this as part of icon utility and not theme. because this changes with icon package change and not theme.